samplers/parser.go

```diff
@@ -601,6 +602,12 @@ func ParseServiceCheck(packet []byte) (*UDPMetric, error) {
 			return nil, errors.New("Invalid service check packet, unrecognized metadata section")
 		}
 	}
+	h := fnv1a.Init32
+	h = fnv1a.AddString32(h, ret.Name)
+	h = fnv1a.AddString32(h, ret.Type)
+	ret.JoinedTags = strings.Join(ret.Tags, ",")
```

Comment:
hm careful here, is the tags slice sorted? the order has to be deterministic in order to get the right hashing

Reply:
It looks like it also needs to be sorted as its doing for `ParseSSFMetric` and `ParseMetric`.